var/spack/repos/builtin/packages/bml/package.py

```diff
@@ -7,20 +7,28 @@
 
 
 class Bml(CMakePackage):
-    """The basic matrix library (bml) is a collection of various matrix data
-    formats (in dense and sparse) and their associated algorithms for basic
-    matrix operations."""
+    """The basic matrix library (bml) is an implementation of various matrix data
+    formats (dense and sparse) and their associated algorithms for basic
+    matrix operations used electronic structure solvers."""
 
     homepage = "https://lanl.github.io/bml/"
-    url = "https://github.com/lanl/bml/tarball/v1.2.2"
+    url = "https://github.com/lanl/bml/archive/refs/tags/v2.2.0.tar.gz"
     git = "https://github.com/lanl/bml.git"
 
-    version("develop", branch="master")
-    version("1.3.1", sha256="17145eda96aa5e550dcbff1ee7ce62b45723af8210b1ab70c5975ec792fa3d13")
-    version("1.3.0", sha256="d9465079fe77210eb2af2dcf8ed96802edf5bb76bfbfdbcc97e206c8cd460b07")
-    version("1.2.3", sha256="9a2ee6c47d2445bfdb34495497ea338a047e9e4767802af47614d9ff94b0c523")
-    version("1.2.2", sha256="89ab78f9fe8395fe019cc0495a1d7b69875b5708069faeb831ddb9a6a9280a8a")
-    version("1.1.0", sha256="29162f1f7355ad28b44d3358206ccd3c7ac7794ee13788483abcbd2f8063e7fc")
+    maintainers = ["jeanlucf22"]
+
+    version("master", branch="master")
```

Comment:
@adamjstewart I know it's been about 4 years, but do you know why the version was `develop` for the `master` branch?

Reply:
> @adamjstewart I know it's been about 4 years, but do you know why the version was `develop` for the `master` branch?

Nvm.  @tgamblin Does your comment about using `develop` instead of `master1 (https://github.com/spack/spack/pull/4213#discussion_r116055678) still apply here?

Reply:
Once upon a time, Spack only considered `develop` to be newer than stable releases, and considered all other string versions to be older than all stable releases. This obviously caused problems, so we used to recommend people call these kinds of versions `develop` instead of `master`. Several years ago, we finally added `master` and several other [blessed strings](https://github.com/spack/spack/blob/v0.19.0/lib/spack/spack/version.py#L64) to the list of versions newer than release versions, so now the correct policy is to keep the version name the same as the branch name unless it isn't in this list.

Reply:
> Once upon a time, Spack only considered `develop` to be newer than stable releases, and considered all other string versions to be older than all stable releases. This obviously caused problems, so we used to recommend people call these kinds of versions `develop` instead of `master`. Several years ago, we finally added `master` and several other [blessed strings](https://github.com/spack/spack/blob/v0.19.0/lib/spack/spack/version.py#L64) to the list of versions newer than release versions, so now the correct policy is to keep the version name the same as the branch name unless it isn't in this list.

Thank you for the clarification Adam!